dashboard/test/controllers/api/v1/pd/workshop_survey_report_controller_test.rb

```diff
@@ -101,20 +101,29 @@ class Api::V1::Pd::WorkshopSurveyReportControllerTest < ::ActionController::Test
     assert_equal Array.new(3, 'Jaime was very funny'), response_hash['this_workshop']['things_facilitator_did_well']
     assert_equal Array.new(3, 'Jaime was very funny'), response_hash['all_my_local_workshops']['things_facilitator_did_well']
 
-    [organizer, admin].each do |user|
-      sign_in(user)
-      @controller = @controller.class.new
-      get :local_workshop_survey_report, params: {workshop_id: workshop_2.id}
-      assert_response :success
-      response_hash = JSON.parse(@response.body)
-
-      assert_equal 1, response_hash['this_workshop']['how_much_learned']
-      assert_nil response_hash['all_my_local_workshops']['how_much_learned']
-      assert_equal({'Jaime' => 1.0}, response_hash['this_workshop']['how_clearly_presented'])
-      assert_nil response_hash['all_my_local_workshops']['how_clearly_presented']
-      assert_equal({'Jaime' => Array.new(3, 'Jaime was very funny')}, response_hash['this_workshop']['things_facilitator_did_well'])
-      assert_nil response_hash['all_my_local_workshops']['things_facilitator_did_well']
-    end
+    sign_in(organizer)
+    @controller = @controller.class.new
```

Comment:
This is kind of a code smell. Why reinitialize `@controller`? It seems like we're trying to force something here.
 [ActionController::TestCase](http://api.rubyonrails.org/v5.1/classes/ActionController/TestCase.html is meant for testing a single controller action. Perhaps this should be an [integration test](http://api.rubyonrails.org/v5.1/classes/ActionDispatch/IntegrationTest.html) that can test multiple controller actions and log in/ log out. 

Really however this looks like it should be split into multiple more tightly scoped test cases. Is signing in and out as different users part of the scenario? Ideally each test case would only assert one thing, or set of similar things that are aspects of the same feature validation.